lib/components/primitive-components/Group/Group.ts

```diff
@@ -631,6 +631,7 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
 
   doInitialPcbTraceRender() {
     const debug = Debug("tscircuit:core:doInitialPcbTraceRender")
+    if (this.getInheritedProperty("circuitJson")) return
```

Comment:
hmm i think its more clear as...

wdyt? you can define that function as `return Boolean(this.getInheritedProperty("circuitJson"))`
```suggestion
    if (this.isPrerendered()) return
```